Makefile

```diff
@@ -1,61 +1,62 @@
-export GO15VENDOREXPERIMENT=1
-
 BENCH_FLAGS ?= -cpuprofile=cpu.pprof -memprofile=mem.pprof -benchmem
-PKGS ?= $(shell glide novendor)
-PKG_FILES ?= *.go example/*.go m3
-LINT_IGNORE = m3/thrift\|thirdparty
+PKGS ?= $(shell go list ./...)
+PKG_FILES ?= $(shell find . -type f -name '*.go' |egrep -v '^./(thirdparty/|vendor/)')
+LINT_IGNORE = (/m3/thrift/|/thirdparty/)
 LICENSE_IGNORE = thirdparty
+GO = GO111MODULE=on go
 
 .PHONY: all
 all: lint test
 
 .PHONY: dependencies
 dependencies:
-	@echo "Installing Glide and locked dependencies..."
-	glide --version || go get -u -f github.com/Masterminds/glide
-	glide install
 	@echo "Installing test dependencies..."
-	go install ./vendor/github.com/axw/gocov/gocov
-	go install ./vendor/github.com/mattn/goveralls
+	$(GO) get github.com/axw/gocov/gocov
+	$(GO) get github.com/mattn/goveralls
 	@echo "Installing golint..."
-	go install ./vendor/github.com/golang/lint/golint
+	$(GO) get -u golang.org/x/lint/golint
```

Comment:
nit: don't need/want `-u` here unless we're going to use it for all of them; don't want some packages being updated while others aren't